gammapy/obs/observation.py

```diff
@@ -43,6 +43,12 @@ class ObservationTable(Table):
     * ``OBS_ID``
     * ``ONTIME``
     * ``LIVETIME``
+    * ``TSTART``
+    * ``TSTOP``
+    * ``AZ``
+    * ``ALT``
+    * ``RA``
+    * ``DEC``
```

Comment:
Link to the format description in the docs (a Sphinx `:ref:` directive should work) instead of trying to duplicate (part of) the information here in this docstring.